test-harness/specs/security-scheme-validation-AC-10.oas3.txt

```diff
@@ -0,0 +1,35 @@
+====test====
+Adding a required header of "Authorization: Bearer <<BEARER_TOKEN>>" results in granted access.
+====spec====
+openapi: 3.0.2
+paths:
+  /todos:
+    get:
+      responses:
+        200:
+          description: OK
+          content:
+            application/json:
+              example: test
+components:
+  securitySchemes:
+    OAuth2:
+      type: oauth2
+      flows:
+        authorizationCode:
+          authorizationUrl: /oauth/authorize
+          tokenUrl: /oauth/token
+          scopes:
+            read: Grants read access
+            write: Grants write access
+            admin: Grants read and write access to administrative information
+security:
+  - OAuth2: [read, write]
+====server====
+mock -p 4010
+====command====
+curl -i http://localhost:4010/todos -H "Authorization: Bearer abc123"
+====expect====
```

Comment:
Why are we sending the same authorization headers for every different type of security Schema? They all work in very different ways: where did these examples come from?

Reply:
Hm.. but don't they all (the three tests) depend on spending `-H "Authorization: Bearer abc123"`? The source of these examples would be me trying to meet the ACs.

Reply:
Bearer is one of many authorization schemes available. I believe the GitHub issue for this links you to the appropriate part of the OpenAPI specification, which points you to https://tools.ietf.org/html/rfc7235#section-5.1

Reply:
For example I don't see the word 'Digest' in this PR, but it's in #425

Reply:
hi @philsturgeon, so there already was an example of `Basic`: https://github.com/stoplightio/prism/blob/271588bc86be61144387ee4e24b39b758f5acec3/test-harness/specs/security-scheme-validation-AC-16_failing.oas2.txt
and I added 'Digest' example: https://github.com/stoplightio/prism/blob/271588bc86be61144387ee4e24b39b758f5acec3/test-harness/specs/security-scheme-validation-AC-17_failing.oas2.txt